src/ceph-volume/ceph_volume/util/system.py

```diff
@@ -33,21 +33,15 @@ def generate_uuid():
 
 def which(executable):
     """find the location of an executable"""
-    locations = (
-        '/usr/local/bin',
-        '/bin',
-        '/usr/bin',
-        '/usr/local/sbin',
-        '/usr/sbin',
-        '/sbin',
-    )
+    path = os.getenv('PATH')
```

Comment:
```suggestion
    path = os.getenv('PATH', '')
```
This makes sure the `split` can be executed in the case of not `PATH` being set. In this case only the current directory is searched.